src/Meilisearch/Meilisearch.csproj

```diff
@@ -17,4 +17,28 @@
     <ItemGroup>
       <PackageReference Include="System.Net.Http.Json" Version="6.0.0" />
     </ItemGroup>
+
+    <ItemGroup>
```

Comment:
Just a curiosity, why do you need to make this change as well? The `partial` keyword doesn't work out of the box?

Reply:
>Just a curiosity, why do you need to make this change as well?

The csproj modification is for the nesting like this

![image](https://user-images.githubusercontent.com/17902415/169009785-70fdfb0a-c66d-4a18-a061-ce2c2e256a69.png)

Without the csproj modification it would looks like this
![image](https://user-images.githubusercontent.com/17902415/169010023-5787cbba-cfa4-4d5c-a17f-7f56e528c42c.png)

>The `partial` keyword doesn't work out of the box?

The `partial` keyword is mandatory if you want to split a class across several files.

Reply:
Oh nice, good to know!